docs/can-guides/commitment/recipes/playlist-editor/6-order.js

```diff
@@ -89,24 +89,26 @@ var PlaylistVM = can.DefineMap.extend("PlaylistVM", {
   }
 });
 
-var Sortable = can.Control.extend({
+can.addJQueryEvents(jQuery);
+
+const Sortable = can.Control.extend({
   "{element} dropmove": function(el, ev, drop, drag) {
     this.fireEventForDropPosition(ev, drop, drag, "sortableplaceholderat");
   },
   "{element} dropon": function(el, ev, drop, drag) {
     this.fireEventForDropPosition(ev, drop, drag, "sortableinsertat");
   },
   fireEventForDropPosition: function(ev, drop, drag, eventName) {
-    var dragData = can.data.get.call(drag.element[0], "dragData");
+    const dragData = can.domData.get(drag.element[0], "dragData");
 
-    var sortables = $(this.element).children();
+    const sortables = $(this.element).children();
 
     for (var i = 0; i < sortables.length; i++) {
       //check if cursor is past 1/2 way
-      var sortable = $(sortables[i]);
+      const sortable = $(sortables[i]);
       if (ev.pageY < Math.floor(sortable.offset().top + sortable.height() / 2)) {
         // index at which it needs to be inserted before
-        $(this.element).trigger({
+        can.domEvents.dispatch(this.element, {
```

Comment:
Using `$.trigger` for dispatching special events through jQuery doesn’t work, I think unless we register them with can-dom-events. I thought it’d be fine to show can-dom-events, but please let me know if there’s a different way we want to teach this.